File: Modul 4/Modul 4 Bahasa python/PRAK404-2410817310007-MuhammadNaufalAbdillah.py
while True:
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    pilihan= int(input("Masukan pilihan : "))

    if pilihan == 5:
        print("Terimakasih, telah mengunakan Kalkulator MUHAMMAD NAUFAL ABDILLAH")
        break
    elif pilihan >= 1  and pilihan <= 5 :

        nilai_1=float(input("Masukan nilai kesatu :"))
        nilai_2=float(input("Masukan nilai kedua :"))
        
        if pilihan == 1:
            hasil=nilai_1+nilai_2
            print(f"Hasil Perjumlahan antara {nilai_1:.2f} dengang {nilai_2:.2f} adalah {hasil:.2f}")
            print("")
        elif pilihan == 2:
            hasil=nilai_1-nilai_2
            print(f"Hasil Pengurangan antara {nilai_1:.2f} dengang {nilai_2:.2f} adalah {hasil:.2f}")
            print("")
        elif pilihan == 3:
             hasil=nilai_1*nilai_2
             print(f"Hasil Perkalian antara {nilai_1:.2f} dengang {nilai_2:.2f} adalah {hasil:.2f}")
             print("")
        elif pilihan == 4:
            hasil=nilai_1/nilai_2
            print(f"Hasil Pembagian antara {nilai_1:.2f} dengang {nilai_2:.2f} adalah {hasil:.2f}")
            print("")
        else:
            print("Input anda salah, silahkan coba lagi")
            print("")
            continue

File: Modul 4/Modul 4 Bahasa C/PRAK404-2410817310007-MuhammadNaufalAbdillah.c
#include <stdio.h>
int main (){
    int pilihan;
    float nilai_1, nilai_2, hasil;

    while (1)
    {
      printf("Pilih Program\n");
      printf("1. Penjumlahan\n2. pengurangan\n");
      printf("3. Perkalian\n4. Pembagian\n");

      printf("5. Exit\nMasukan pilihan  :");
      scanf("%d", &pilihan);

    if (pilihan== 5)
    {
     printf("Teimakasih, telah mengunakan Kalkulator MUHAMMAD NAUFAL ABDILLAH");
     break;
    }

    if (pilihan<1||pilihan>5)
    {
     printf("Input anda salah, silahkan mencoba lagi\n");
     continue;
    }

    if (pilihan>=1&&pilihan<=4)
    {
     printf("Masukan nilai kesatu :");
     scanf("%f", &nilai_1);
     printf("Masukan nilai kedua :");
     scanf("%f", &nilai_2);
    }

    switch (pilihan)
    {
    case 1:
       hasil=nilai_1=nilai_2;
       printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f \n \n", nilai_1, nilai_2, hasil);
        break;
    case 2:
        hasil=nilai_1-nilai_2;
        printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f \n \n", nilai_1, nilai_2, hasil); 
        break;
    case 3:
        hasil=nilai_1*nilai_2;
        printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f \n \n", nilai_1, nilai_2, hasil);
        break;
    case 4:
            if(nilai_2!=0 || nilai_1!=0){
        hasil=nilai_1/nilai_2;
        printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f \n \n", nilai_1, nilai_2, hasil);
       } else{printf("Pembagian nol tidak bisa\n\n");}
        break;
    }
    
    }
    return 0;
}